### ISSUE TITLE
Fleeing enemies don't count as dead.

### ISSUE THREAD

**Issue by ohrrpgce-bugbot** [ohrrpgce/ohrrpgce#241]
[bz#241]

While playing JSH's Query, a game made ages ago, I ran into battles where the foes are supposed to flee when you kill them; however, the dead enemies don't leave the screen. Instead, the hang on the edge, allowing you to attack them further and they themselves can also attack. I'm not sure if this is the game's fault or game.exe's. But I've provided a screenshot.

http://www.geocities.com/twinhamster/whatami4.bmp

As seen in the screenshot, the reversed image of the enemy appears on the side of the screen, and is still targetable.

From: **Ken &lt;twinhamster&gt;**
Reported version: 20060803 Hasta-la-qb+
Operating system:   Windows XP
Duplicates:   bz⁠#184

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @rversteegen

This sounds to me like it's probably a duplicate of the very awful bug bz⁠#184

**Comment by ohrrpgce-bugbot** [comment]
Comment author: **Ken &lt;twinhamster&gt;**

Well, yes, but bug bz⁠#184 is simply a case where a multi-hit attack must be completed before an enemy can count as dead. In this case, the enemy is alive and well (with very little health).

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @rversteegen

Huh? I don't read bug bz⁠#184 as having anything to do with multihit attacks. That bug report doesn't describe the killed enemy continuing to attack, but in both cases its a case of dead enemies not dying.

What do you mean by "very little health"?

**Comment by ohrrpgce-bugbot** [comment]
Comment author: **Ken &lt;twinhamster&gt;**

I must have bug bz⁠#184 confused with another one then...

I've seen two different behaviors for the killed enemies. The enemy will either die permanently once it is hit again, or it will be able to withstand multiple hits without dying. The battle always ends when the rest of the enemies are killed.

In the battle shown, enemies should die in one hit (Since the hero deals about 3000 damage), so I find it odd that the enemy can take multiple, separate hits from the hero.

**Comment by ohrrpgce-bugbot** [comment]
Comment author: **Ken &lt;twinhamster&gt;**

Oh. Nevermind, this is the same situation as bug bz⁠#184. Sorry.

\*\*\* This bug has been marked as a duplicate of bug bz⁠#184 \*\*\*
